### PR TITLE
Fixed Marker Fetching

### PR DESCRIPTION
**Pull request by Im-Danny-Q**
- Markers are now automatically populated from the backend 

- TODO:
- Finish code for pushing markers to database
- Fix code for adding new markers on click (Or we can just add a field for lat and long manually. (Maybe we can autopopulate a text box on click instead of adding a marker?)
- Add image functionality to backend/POST request

### CODE REVIEW

**Comment by michaelBelsanti** [comment]
Move image into assets instead of src

File: src/map_utils.js [hunk 1]
```diff
@@ -236,37 +275,43 @@ function handleSubmit() {
...
+//       console.log(markerData.latitude)
+//       console.log(markerData.longitude)
+      
+//       placeMarker(markerPosition, map, 'Custom Marker', 'TESTING');
+//     })
+//   })
+//   .catch((error) => {
+//     console.error('Error:', error);
+//   });
+// }
```
[the remark this answers is not in the document]
Comment (Im-Danny-Q) [reply src/map_utils.js]: yeah but that's scary

File: src/map_utils.js [hunk 2]
```diff
@@ -1,124 +1,163 @@
...
+//     'Error: The Geolocation service failed.' :
+//     'Error: Your browser doesn\'t support geolocation.';
+//   alert(error);
+//   if (initialLocation) {
+//     map = new google.maps.Map(document.getElementById('map'), {
+//       center: initialLocation,
+//       zoom: 14,
+//     });
+//   }
+// };
```
[the remark this answers is not in the document]
Comment (Im-Danny-Q) [reply src/map_utils.js]: I thought it was breaking stuff (its not)

File: src/map_utils.js [hunk 3]
```diff
@@ -1,124 +1,163 @@
...
+
+var map;
+var markers = [];
+var locationInput = document.getElementById('location');
+var setMarkerMode = false;
 
 export function initMap() {
-  if (navigator.geolocation) {
-    navigator.geolocation.getCurrentPosition(function(position) {
-      var initialLocation = new google.maps.LatLng(position.coords.latitude, position.coords.longitude);
```
[the remark this answers is not in the document]
Comment (Im-Danny-Q) [reply src/map_utils.js:6]: yeah but i couldn't figure out scope and stuff so i just hardcoded it

**Comment by Im-Danny-Q** [comment]
Adjusted for mike's comments

**Pull request merged**